x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/total_feature_importance_summary/feature_importance_summary.tsx

```diff
@@ -0,0 +1,243 @@
+              sortedData.push({
+                featureName: feature.feature_name,
+                meanImportance: featureClass.importance.mean_magnitude,
+                className: featureClass.class_name,
+              });
+            });
+        });
+        _barSeriesSpec = {
+          xAccessor: 'featureName',
+          yAccessors: ['meanImportance'],
+          splitSeriesAccessors: ['className'],
+          stackAccessors: ['featureName'],
+        };
+      }
+    }
+    // if regression
+    if (isRegressionTotalFeatureImportance(totalFeatureImportance[0])) {
+      classificationType = 'regression';
+
+      sortedData = (totalFeatureImportance as RegressionTotalFeatureImportance[]).map(
+        (d: RegressionTotalFeatureImportance) => ({
+          featureName: d.feature_name,
+          meanImportance: d.importance.mean_magnitude,
+        })
+      );
+    }
+
+    // sort from largest importance at top to smallest importance at bottom
+    sortedData = sortedData.sort((a, b) => b.meanImportance - a.meanImportance);
+
+    // only show legend if it's a multiclass
+    const _showLegend = classificationType === 'multiclass_classification';
+    const _chartHeight =
+      totalFeatureImportance.length * (totalFeatureImportance.length < 5 ? 40 : 20) + 50;
+    return [sortedData, _barSeriesSpec, _showLegend, _chartHeight];
+  }, [totalFeatureImportance]);
+
+  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
+  const tickFormatter = useCallback((d) => Number(d.toPrecision(3)).toString(), []);
+
+  return (
+    <EuiPanel>
+      <div>
+        <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize="xs">
+              <EuiTitle size="xs">
+                <span>
+                  <FormattedMessage
+                    id="xpack.ml.dataframe.analytics.exploration.featureImportanceSummaryTitle"
+                    defaultMessage="Overall feature importance"
+                  />
+                </span>
+              </EuiTitle>
+              <EuiFlexItem grow={false}>
+                <EuiIconTip content={tooltipContent} />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiSpacer />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              target="_blank"
+              iconType="help"
+              iconSide="left"
+              color="primary"
+              href={`${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-feature-importance.html`}
+            >
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.exploration.featureImportanceDocsLink"
+                defaultMessage="Feature importance docs"
+              />
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </div>
+      <Chart
+        className="story-chart"
```

Comment:
I assume this `className` was copied from the examples and is not needed?

Reply:
Thanks for catching that. Updated here https://github.com/elastic/kibana/pull/78238/commits/e18bc414d34f7ca2611b2fa0cbcf0a2403d980a4